scapy/contrib/ife.uts

```diff
@@ -0,0 +1,27 @@
+% IFE test campaign
+
+#
+# execute test:
+# > test/run_tests -P "load_contrib('ife')" -t scapy/contrib/ife.uts
+#
+
++ Basic layer handling
+= build basic IFE frames
+
+frm = Ether()/IFE(tlvs=[IFESKBMark(), IFETCIndex()])
+frm = frm.build()
+frm = Ether(frm)
+
+= add padding if required
+
+frm = Ether()/IFE(tlvs=[IFETCIndex()])
+assert(len(raw(frm)) == 24)
+
+frm = Ether()/IFE(tlvs=[IFESKBMark(), IFETCIndex()])
+assert(len(raw(frm)) == 32)
+
+= variable payload
+
+frm = Ether()/fuzz(IFE(tlvs=[IFETlvStr()]))
+frm = frm.build()
+frm = Ether(frm)
```

Comment:
Here it would be great if you have an assert testing `frm`. You should use `Ether(src="aa:aa:aa:aa:aa:aa", dst="bb:bb:bb:bb:bb:bb")` to ensure it stays correct on all computers